recipes/conda-index/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "conda-index" %}
+{% set version = "0.2.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/conda/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: b3ca5248a714d355dff29b1c07f4f5b3088e9f660815b19bb40f0379664c744c
+
+build:
+  script: {{ PYTHON }} -m pip install --no-build-isolation . -vv
+  number: 0
+  noarch: python
+
+requirements:
+  host:
+    - flit-core
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - click >=8
+    - conda >=4.12.0
+    - conda-package-streaming
+    - filelock
+    - jinja2
+    - more-itertools
+    - PyYAML >=6
+
+test:
+  requires:
+    - conda-build >=3.21.0
+    - conda-package-handling >=1.9.0
+    - coverage  # [toml]
```

Comment:
Could you please clarify what this selector is doing?

Reply:
This is a mistake. Coverage's toml support should be installed.

Reply:
Gotcha. So this is related to issue ( https://github.com/conda/conda/issues/2984 ) and a couple others ( https://github.com/conda/conda/issues/2984#issuecomment-1333274998 )

In any event, we don't currently have this exposed. Given this is [just `tomli`]( https://github.com/nedbat/coveragepy/blob/7e7c44b4f5c484105559690b1efccd84839bc640/setup.py#L112 ), maybe we can list that here too?

Reply:
Just to be clear, you're suggesting remove the ` # [toml]` selector and adding `tomli` as an additional test requirement?

Reply:
yes

Reply:
https://github.com/nedbat/coveragepy/blob/master/setup.py#L112